docs/settings.md

```diff
@@ -0,0 +1,164 @@
+# Configuration
+
+uMap runs with Django, so any Django setting should work, if you know what you
+are doing.
+
+The Django settings reference is here: https://docs.djangoproject.com/en/4.2/ref/settings/
+
+Here are a few relevent settings for uMap.
+
+
+#### ALLOWED_HOSTS
+
+The hosts that uMap expects.
+`ALLOWED_HOSTS = ['umap.mydomain.org']`
+
+Can be set through env var too: `ALLOWED_HOSTS=umap.mydomain.org`
+
+#### DEBUG
+
+Set it to True for easier debugging in case of error.
+
+#### EMAIL_BACKEND
+
+Must be configured if you want uMap to send emails to anonymous users.
+
+See [Emails](install.md#emails) for more details.
+
+#### ENABLE_ACCOUNT_LOGIN
+
+Do you want users to be able to create an account directly on your uMap instance
+(instead of only using OAuth).
+
+Can be set through env var: `ENABLE_ACCOUNT_LOGIN=1`
+
+#### FROM_EMAIL
+
+See `EMAIL_BACKEND`.
+
+#### LANGUAGE_CODE
+
+Set it to the default language you want. `LANGUAGE_CODE = "it"`
+
+#### LEAFLET_LONGITUDE, LEAFLET_LATITUDE, LEAFLET_ZOOM
+
+Default longitude, latitude and zoom for the map
+
+#### MEDIA_ROOT
+
+Where uMap should datalayers and icons, must be consistent with your
+Nginx configuration.
+
+#### SECRET_KEY
+
+Must be defined to something unique and secret.
+
+#### SITE_URL
+
+The final URL of you instance, including the protocol:
+
+`SITE_URL="http://umap.org"`
+
+
+#### SHORT_SITE_URL
+
+If you have a short domain for sharing links.
+
+Eg.: `SHORT_SITE_URL=https://u.umap.org`
+
+
+#### SITE_NAME
+
+The name of the site, to be used in header and HTML title.
+
+
+#### STATIC_ROOT
+
+Where uMap should store static files (CSS, JS…), must be consistent with your
```

Comment:
```suggestion
Where uMap should find your static files (CSS, JS…), must be consistent with your
```

Reply:
Not sure about this change: when running `collectstatic`, Django will cp static to that STATIC_ROOT directory. And Django/uMap itself should not server static files, as you know for sure :)

Reply:
You're right, for these standard settings I would redirect to the Django documentation.